app/src/main/java/com/yalta/activities/fragments/AdminBrowseFragment.kt

```diff
@@ -6,22 +6,26 @@ import androidx.fragment.app.Fragment
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.view.inputmethod.EditorInfo
+import androidx.core.widget.addTextChangedListener
 import androidx.databinding.DataBindingUtil
 import androidx.lifecycle.ViewModelProvider
 import androidx.recyclerview.widget.DividerItemDecoration
 import com.yalta.R
 import com.yalta.databinding.FragmentAdminBrowseBinding
+import com.yalta.utils.ViewUtils.hideKeyboard
 import com.yalta.viewmodel.AdminBrowseViewModel
+import kotlinx.android.synthetic.main.fragment_admin_browse.*
 
 class AdminBrowseFragment : Fragment() {
     private val viewModel by lazy { ViewModelProvider(this).get(AdminBrowseViewModel::class.java) }
+    lateinit var binding: FragmentAdminBrowseBinding
```

Comment:
if it's `lateinit`, shouldn't it be `val`?

Reply:
`lateinit` can only be `var`)